enterprise/migrations/0001_initial.py

```diff
@@ -0,0 +1,125 @@
+            name='enterprisecustomermember',
+            unique_together=set([('enterprise_customer', 'user_id')]),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='short_code',
+            field=autoslug.fields.AutoSlugField(editable=True,
+                                                help_text='Short code will be used to reference this Enterprise Customer in other parts of the system,i.e. SSO, ecommerce app, analytics etc.',
+                                                max_length=120, populate_from='name', unique=True),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='short_code',
+            field=autoslug.fields.AutoSlugField(blank=True, editable=True,
+                                                help_text='Short code will be used to reference this Enterprise Customer in other parts of the system,i.e. SSO, ecommerce app, analytics etc.',
+                                                max_length=120, populate_from='name', unique=True),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='short_code',
+            field=autoslug.fields.AutoSlugField(blank=True, editable=True,
+                                                help_text='Short code will be used to reference this Enterprise Customer in other parts of the system,i.e. SSO, ecommerce app, analytics etc. It is allowed to override it at creation, but it shouldnever be edited on existing records, as it might break integration with other apps.3',
+                                                max_length=120, populate_from='name', unique=True),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='short_code',
+            field=autoslug.fields.AutoSlugField(editable=True,
+                                                help_text='Short code will be used to reference this Enterprise Customer in other parts of the system,i.e. SSO, ecommerce app, analytics etc. It is allowed to override it at creation, but it shouldnever be edited on existing records, as it might break integration with other apps.3',
+                                                max_length=120, populate_from='name', unique=True),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='short_code',
+            field=autoslug.fields.AutoSlugField(blank=True, editable=True,
+                                                help_text='Short code will be used to reference this Enterprise Customer in other parts of the system,i.e. SSO, ecommerce app, analytics etc. It is allowed to override it at creation, but it should never be edited on existing records, as it might break integration with other apps.',
+                                                max_length=120, populate_from='name', unique=True),
+        ),
```

Comment:
@mulby The plan is to squash all the migrations into one, so in final version there won't be that much of alterations.